pkg/api/controllers/swagger.yaml

```diff
@@ -456,6 +456,8 @@ components:
           type: string
         err:
           type: string
+        txs:
+          $ref: '#/components/schemas/CreateTransactionResponse'
```

Comment:
CreateTransactionResponse represents the following structure : 
```
{
   "data": [ .... transactions ]
}
```
Seems not the good one for this case. The '#/components/schemas/TransactionCommitError" represent the good one (is error is properly translated, see previous comment).